### ISSUE TITLE
Hreflang tag shouldn't be outputted when a translation is connected but unpublished

### ISSUE THREAD

**Issue by nowori** [lloc/Multisite-Language-Switcher#173]
Hey there,

Currently, MSLS outputs the homepage URL for hreflang tags, when a page is connected but not published. This seems to be bad practice, which makes sense as hreflang tags are supposed to be reciprocal, that is they should all reference each other on their respective page, which doesn't happen when we output the homepage hreflang on a specific page = it wouldn't be reciprocated on the homepage.

Difficult to find references on the topic but here are some:
https://webmasters.stackexchange.com/questions/118439/use-of-rel-alternate-on-multilingual-site/118455
https://cognitiveseo.com/blog/17150/multi-language-website-mistakes/

I am not proposing a pull request as I'm unsure how to best cater for this.

It seems that the goal of this code was to implement this scenario:
https://github.com/lloc/Multisite-Language-Switcher/blob/d5747af1cb86d0a5ae34ad306fabc311b91195a6/includes/MslsOutput.php#L110-L112

But it seems it is never `null` because of:
https://github.com/lloc/Multisite-Language-Switcher/blob/d5747af1cb86d0a5ae34ad306fabc311b91195a6/includes/MslsOptions.php#L233

But I guess this is not where this should be fixed as it is probably used to output the language switcher, amongst other things. Would appreciate if you could have a look.

For now and if others want to fix this behavior, this is how I've implemented it:

```php
add_filter( 'mlsl_output_get_alternate_links', function( $url, $blog ) {

	if ( $url === trailingslashit( $blog->siteurl ) )
		return null;
	else
		return $url;

}, 10, 2 );
```

Thank you

**Comment by apermo** [comment]
I agree that unpublished content should not create any output. I'll have a look into this, as this might be an upcoming issue for me, and if I can reproduce it, I'll open a PR.
